app/views/passengers/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Edit passenger</h1>
+
+<%= render partial: 'layouts/error_messages',
```

Comment:
I like the way you used a partial for the error messages. It would be interesting to also use a partial for the rest of the form.